x/concentrated-liquidity/position.go

```diff
@@ -219,10 +219,10 @@ func (k Keeper) GetUserPositionsSerialized(ctx sdk.Context, addr sdk.AccAddress,
 	var prefix []byte
 	var expectedKeyPartCount int
 	if poolId == 0 {
-		expectedKeyPartCount = 3
+		expectedKeyPartCount = 2
 		prefix = types.KeyUserPositions(addr)
 	} else {
-		expectedKeyPartCount = 2
+		expectedKeyPartCount = 1
```

Comment:
This is because I removed the extraneous separator that was in the key

Reply:
Didn't we add a extraneous separator thus should be incremented? https://github.com/osmosis-labs/osmosis/pull/5470/files#diff-36982b4b327909ec37e5cba27c20f24dd4b6f1fc1fa230561ed4718384db68adR189

Reply:
no thats added to the prefix, so its not in the key that is getting matched here